aldryn_django/startup.py

```diff
@@ -18,11 +18,10 @@ def wsgi(path):
     from django.conf import settings
     from dj_static import Cling, MediaCling
     app = get_wsgi_application()
-    if settings.ENABLE_SYNCING:
-        if not settings.STATIC_URL_IS_ON_OTHER_DOMAIN:
-            app = Cling(app)
-        if not settings.MEDIA_URL_IS_ON_OTHER_DOMAIN:
-            app = MediaCling(app)
+    if not settings.STATIC_URL_IS_ON_OTHER_DOMAIN:
```

Comment:
This code shall all be removed. You can probably also remove `dj_static` from the requirements then.

Reply:
removed